Multisig/Data/Services/Safe Client Gateway Service/ChainsListRequest.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ChainsRequest.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/23/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct ChainsListRequest: JSONRequest {
+    var httpMethod: String {
+        "GET"
+    }
+
+    var urlPath: String {
+        "/v1/chains"
+    }
+
+    typealias ResponseType = [SCGModels.Chain]
+}
+
+extension SafeClientGatewayService {
+    @discardableResult
+    func chains(completion: @escaping (Result<ChainsListRequest.ResponseType, Error>) -> Void) -> URLSessionTask? {
```

Comment:
I would be for continuing our tradition of adding `async` prefix for all async calls

Reply:
I remember that @DmitryBespalov  wanted to remove async from now on because all our requests are asyc so no meaning of having it everywhere

Reply:
let's do async for now and change it in a separate refactor